sql/src/test/java/io/crate/testing/CollectingRowReceiver.java

```diff
@@ -94,10 +94,11 @@ public Bucket result() throws Exception {
         }
     }
 
-    private static class PausingReceiver extends CollectingRowReceiver {
+    public static class PausingReceiver extends CollectingRowReceiver {
 
         private final int pauseAfter;
         private int numRows = 0;
+        public boolean isFinished = false;
```

Comment:
I think isFinished could be added to the CollectingRowReceiver and then the PausingReceiver can be made private again. If that isn't possible either the static factory method should be removed or this ctor should be protected/private. 2 ways to create the class aren't really necessary